app/controllers/infra_networking_controller.rb

```diff
@@ -266,6 +266,7 @@ def default_node
     options = {:model => "Switch", :named_scope => :shareable}
     process_show_list(options) if @show_list
     @right_cell_text = _("All Switches")
+    @right_cell_text += _(" (Names with \"%{search_text}\")") % {:search_text => @search_text} if @search_text
```

Comment:
```
@right_cell_text = @search_text ? _(" (Names with \"%{search_text}\")") % {:search_text => @search_text} : _("All Switches")
```

Or an `if ... else` statement.

I think that would be more readable and easier to follow that assigning and then reassigning the variable.

Reply:
Yes, I can make it more readable. Just a little note: we need _"All Switches"_ to be in `@right_cell_text` in both cases. If `@search_text` is set, then _"(Names with "...")"_ should be just added to _"All Switches"_.

Reply:
That's one more argument to do the `if`. Because generally we do not want co concatenate human readable strings. That limits the translators.

Reply:
What about 12 other occurences of `@right_cell_text += ...` in _manageiq-ui-classic_ ?

Reply:
IMO should be fixed too.

WDYT, @mzazrivec ?

Reply:
Probably yes.

Reply:
After some investigation I've found that it is not easy to change other occurrences of `@right_cell_text += ...` because they are located in such places in the code where we don't know exactly what will be the original `@right_cell_text`. It depends on the model/accordion we access and other details. There are more possibilities, I would have to add a lot of extra conditions to cover all possibilities. I'm not sure if it was better.